code401Challenges/src/test/java/code401Challenges/QuickSortTest.java

```diff
@@ -0,0 +1,53 @@
+package code401Challenges;
+
+import com.sun.xml.internal.bind.v2.model.annotation.Quick;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class QuickSortTest {
+
+    @Test
+    public void testMergeSort_unevenSize_Positive(){
```

Comment:
While reusing the merge sort tests was a great time-saver, you should have updated the name of the test functions so they don't all say merge sort